docs/modules/maintain-cluster/pages/enterprise-rest-api.adoc

```diff
@@ -0,0 +1,417 @@
+            <key-store-provider>SUN</key-store-provider>
+            <trust-store>/path/to/truststore</trust-store>
+            <trust-store-password>myTrustStorePassword</trust-store-password>
+            <trust-store-type>JKS</trust-store-type>
+            <trust-store-provider>SUN</trust-store-provider>
+            <enabled-protocols>TLSv1.2, TLSv1.3</enabled-protocols>
+            <protocol>TLS</protocol>
+            <certificate>/path/to/certificate</certificate>
+            <certificate-key>/path/to/certificate-key</certificate-key>
+            <trust-certificate>/path/to/trust-certificate</trust-certificate>
+            <trust-certificate-key>/path/to/trust-certificate-key</trust-certificate-key>
+        </ssl>
+    </rest>
+</hazelcast>
+----
+--
+
+YAML::
++
+[source,yaml]
+----
+hazelcast:
+  rest:
+    enabled: true
+    ssl:
+      enabled: true
+      client-auth: NEED
+      ciphers: TLS_RSA_WITH_AES_128_CBC_SHA, TLS_RSA_WITH_AES_128_CBC_SHA256
+      enabled-protocols: TLSv1.2, TLSv1.3
+      key-alias: myKeyAlias
+      key-password: myKeyPassword
+      key-store: /path/to/keystore
+      key-store-password: myKeyStorePassword
+      key-store-type: JKS
+      key-store-provider: SUN
+      trust-store: /path/to/truststore
+      trust-store-password: myTrustStorePassword
+      trust-store-type: JKS
+      trust-store-provider: SUN
+      protocol: TLS
+      certificate: /path/to/certificate
+      certificate-key: /path/to/certificate-key
+      trust-certificate: /path/to/trust-certificate
+      trust-certificate-key: /path/to/trust-certificate-key
+----
+====
+
+
+== JWT Based Authorization
+
+After a successful authentication by making request to token endpoint (see the <<obtaining-a-token, Obtaining a Token section>> for details), you will receive a short-living JWT token with assigned role names as a claim for authorization checks in subsequent REST calls.
+
+Token issued by one member is not trusted by other members.
+
+Tokens are valid for 15 minutes by default. To change it you can use the rest configuration as shown in the below declarative example:
+
+[tabs]
+====
+XML::
++
+--
+[source,xml]
+----
+<hazelcast>
+    <rest enabled="true">
+        <token-validity-seconds>300</token-validity-seconds>
+    </rest>
+</hazelcast>
+----
+--
+
+YAML::
++
+[source,yaml]
+----
+hazelcast:
+  rest:
+    enabled: true
+    token-validity-seconds: 300
+----
+====
+
+[#obtaining-a-token]
+== Obtaining a Token
+
+To obtain a token, you should send a `POST` request to the token endpoint at `/hazelcast/rest/api/v1/token`.The request should include the `username` and `password` in JSON format.If authentication successful, the response will contain a token.
+
+Example request using cURL:
+
+[source,shell]
+----
+curl -X 'POST' \
+  'http://localhost:8443/hazelcast/rest/api/v1/token' \
+  -H 'Content-Type: application/json' \
+  -d '{
+  "username": "restuser",
+  "password": "restpassword"
+}'
+----
+
```

Comment:
can you also add the example respone?

Reply:
fixed at https://github.com/hazelcast/hz-docs/pull/1097/commits/e4c03741e15cfb67bfa7ce377a0dbb1f94bfab7a